pkg/apis/operators/v1alpha1/servicebinding_types.go

```diff
@@ -19,13 +19,13 @@ type ServiceBindingSpec struct {
 	// +optional
 	MountPath string `json:"mountPath,omitempty"`
 
-	// EnvVarPrefix is the prefix for environment variables
+	// NamePrefix is the prefix for environment variables or file name
 	// +optional
-	EnvVarPrefix string `json:"envVarPrefix,omitempty"`
+	NamePrefix string `json:"namePrefix,omitempty"`
 
-	// Custom env variables
+	// Custom mappings
 	// +optional
-	CustomEnvVar []corev1.EnvVar `json:"customEnvVar,omitempty"`
+	Mappings []corev1.EnvVar `json:"mappings,omitempty"`
```

Comment:
We should change the type here: `corev1.EnvVar` should not be the type we use - it should be something similar to https://github.com/k8s-service-bindings/spec/blob/master/internal/service.binding/v1alpha2/service_binding.go#L69

Reply:
+1

Reply:
done